webserver/Pages/Manage/ClientCRUD.cs

```diff
@@ -48,9 +48,10 @@ public class ClientCRUD : PageModel {
             }
         }
     }
-}
-public class ClientINFO{
-    public string Id { get; set; } = string.Empty;
-    public string UserName { get; set; } = string.Empty;
-    public string Occupation{get; set; } = string.Empty;
+
+    public class ClientINFO{
+        public string Id { get; set; } = string.Empty;
+        public string UserName { get; set; } = string.Empty;
+        public string Occupation{get; set; } = string.Empty;
```

Comment:
```suggestion
        public string Occupation{ get; set; } = string.Empty;
```

You need to setup an auto formatted in your IDE.